test/fixtures.yml

```diff
@@ -0,0 +1,44 @@
+usps:
```

Comment:
I don't understand how this file is used? Where did all this information come from?